src/botPage/bot/Interface/MiscInterface.js

```diff
@@ -29,7 +29,8 @@ export default Interface =>
                 notifyTelegram: this.notifyTelegram,
                 getTotalRuns  : () => this.tradeEngine.getTotalRuns(),
                 getBalance    : type => this.tradeEngine.getBalance(type),
-                getTotalProfit: () => this.tradeEngine.getTotalProfit(),
+                getTotalProfit: (...args) =>
```

Comment:
We're not passing multiple arguments here, so for clarity you could change `...args` into `toString`.

```suggestion
                getTotalProfit: (toString) =>
```